client/src/store/mutations.js

```diff
@@ -29,4 +29,9 @@ export default {
         logger.debug("Set Friends:", friends)
         state.friends = friends
     },
+
+    clearFriendList(state, { friends }) {
```

Comment:
If clear friends list just empties the list then why does it need an argument? Get rid of the friends argument.

Reply:
ok